tests/people/edit/query.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var test = require('tape');
+var query = require('../../../lib/queries/people/edit.js');
+
+var fields = ['first_name', 'last_name', 'job_title', 'phone']; // should add email to this too for Admins?
```

Comment:
It's better to add the questions on the issue. The plugin is independent to the main application so we don't have any admins or primary. The role of the function is to update some fields. Then it's the responsibility of the code where you call this function to pass the right fields.